lib/App/MintTag.pm

```diff
@@ -105,6 +105,36 @@ sub mint_tag ($self, $auto_mode = 0) {
   };
 }
 
+# Tag an arbitrary sha according to the config file. This bails unless there is
+# exactly one step in config with a defined tag_prefix.
+sub tag_arbitrary_sha ($self, $sha) {
+  my $num_prefixes = 0;
+  my $tagging_step;
+
+  for my $step ($self->config->steps) {
+    if ($step->tag_prefix) {
+      $num_prefixes++;
+      $tagging_step = $step;
+    }
+  }
+
+  unless ($num_prefixes == 1) {
+    die "refusing to tag sha; config file is ambiguous about how to tag.\n";
```

Comment:
No capitalization, so it's not a proper sentence, but then the full stop at the end.  I just don't know the rules anymore.

Reply:
I like to drop these in every once in a while to distract you from all the back doors.